api/company-user/index.js

```diff
@@ -41,6 +41,32 @@ exports.activationStatus = async (ctx) => {
   };
 };
 
+exports.changePassword = async (ctx) => {
+  const {
+    newPassword, currentPassword
+  } = ctx.request.body.passwords;
+
+  let { user } = ctx;
```

Comment:
```suggestion
  const { user } = ctx;
```
Always use const unless you reassign the var.

Reply:
🏁